src/ChatApp/chatapp.client/src/api/api.ts

```diff
@@ -0,0 +1,355 @@
+import { chatHistorySampleData } from '../constants/chatHistory'
+
+import { ChatMessage, Conversation, ConversationRequest, CosmosDBHealth, CosmosDBStatus, UserInfo } from './models'
+
+// TODO: This is where we will redirect to the Azure Function instead of existing backend for chat response
```

Comment:
this comment is no longer relevant. already calling into our new .net api backend from here